tests/test_ipynb.py

```diff
@@ -342,23 +333,17 @@ def test_empty_string() -> None:
 
 
 def test_unparseable_notebook() -> None:
-    msg = (
-        r"File 'tests[/\\]data[/\\]notebook_which_cant_be_parsed\.ipynb' "
-        r"cannot be parsed as valid Jupyter notebook\."
-    )
+    path = DATA_DIR / "notebook_which_cant_be_parsed.ipynb"
+    msg = rf"File '{path}' cannot be parsed as valid Jupyter notebook\."
```

Comment:
this didn't pass when I tried running the CI checks in a fork, I had to use
```
'{re.escape(str(path))}'
```
instead of
```
'{path}'
```

https://github.com/MarcoGorelli/black/pull/1